projection/core/src/multi-jvm/scala/com/lightbend/lagom/internal/projection/ProjectionRegistrySpec.scala

```diff
@@ -172,28 +172,37 @@ class ProjectionRegistrySpec
 
       val testProbe = registerProjection(projectionName, tagNames.toSet)
       testProbe.ignoreMsg {
-        case _ => true
+        case FakeStarting(`tagName001`) => false
+        case _                          => true
       }
 
+      // get FakeStarting message from tagName001
+      expectMsgFromWorker(
+        FakeStarting(tagName001),
+        "do-pause-and-resume-worker-test-expect-starting",
+        testProbe,
+        multiExpectTimeout
+      )
+
       // await until seen as ready
       expectWorkerStatus(projectionName, tagName001, Started)
```

Comment:
If the problem is that stopping/Starting `tag001` may be too fast and the probe gets mixed signals from other tags, can't we just:

```suggestion
       expectWorkerStatus(projectionName, tagName001, Started) 
       expectWorkerStatus(projectionName, tagName002, Started) 
       expectWorkerStatus(projectionName, tagName003, Started) 
       expectWorkerStatus(projectionName, tagName004, Started) 
       expectWorkerStatus(projectionName, tagName005, Started) 
```

So no stopping/Starting happens until we're 100% certain all workers are ready? This way we're certain the only signals observed in the probe will be caused by the4 test code (stop/start `tag001`).

Reply:
I mean, that suggestion should be enough to fix the issue, no?

Reply:
Correction, the above suggestion doesn't compile:

```suggestion
    tagNames.foreach(tagName => 
        expectWorkerStatus(projectionName, tagName, Started)
      )
```

This one does. :-D

Reply:
WDYT?

Reply:
I implemented that like that before and move out. The reason is that all those other tags are just a distraction in the test. 

The goal of the test is to prove that if you have a projection, you can stop and restart using the name and a tag. We may want to check that the other tags didn't stop after we send the command for tagName001, but I would avoid any unnecessary cluttering

Reply:
I think those other tags are necessary.

The important thing is the test asserts that after stopping/Starting, only on worker is restarted (meaning no other worker was restarted.

If we ignore all messages from workers 2,3,4, and 5 we won't detect a bug where the request to stop `tag001` actually stops all workers. 

It's an implicit assertion because we can't assert over something that doesn't happen, but ignoring all signals from other workers we're definitely missing the chance to detect undesired state changes.